content/en/dashboards/guide/semantic_colors.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Semantic color guide
+kind: guide
```

Comment:
I can build on this later, this is a good placeholder for the work you did on the table! Adding the private so it's not searchable until it's ready to go

```suggestion
kind: guide
private: true
```